src/App.jsx

```diff
@@ -14,70 +16,71 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
+  const clearSelection = () => setSelectedGood('');
+  const isSelected = good => selectedGood === good;
```

Comment:
Redundant function
you can check like this one 
```
if (selectedGood) {
  // do something if selected
}
```
in case empty string it fill be falsy